cli/command/system/version.go

```diff
@@ -17,7 +16,7 @@ import (
 
 var versionTemplate = `Client:
  Version:      {{.Client.Version}}
- API version:  {{.Client.APIVersion}}
+ API version:  {{.Client.ActualAPIVersion}} (default version {{.Client.DefaultAPIVersion}})
```

Comment:
When I wrote not consistent I meant that when there is no downgrade it outputs "1.26" but when it downgrades it prints out "1.24 (downgraded from 1.26)" and when you are using `--format` to grab actual values in code that makes it challenging (and inconsistent) when .Client.APIVersion can be in a variety of formats.